app/src/main/java/com/musagenius/ocrapp/data/utils/StorageManager.kt

```
@@ -0,0 +1,209 @@
+            Log.e(TAG, "Error getting total storage", e)
+            0L
+        }
+    }
+
+    /**
+     * Get used storage in megabytes
+     */
+    fun getUsedStorageMB(): Long {
+        return try {
+            getTotalStorageMB() - getAvailableStorageMB()
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)